about.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title> About</title>
+  </head>
+  <ul>
+<li><a href="portfolio.html">Portfolio</a></li>
+<li><a href="hobby-blog.html">Blog</a></li>
+<li><a href="about.html">About</a></li>
+</ul>
```

Comment:
Watch your indentation - you should have your editor set up to do this for you automatically.